api/processors/actions/teacher/left_project_prematurely.py

```diff
@@ -21,7 +22,7 @@ def _update_groups(self) -> None:
         self.teacher.groups.clear()
 
     def _set_statuses(self) -> None:
-        self.teacher.project_status = TeacherLogEventType.LEFT_PREMATURELY
+        self.teacher.project_status = TeacherProjectStatus.LEFT_PREMATURELY
```

Comment:
Yes, the tests are passed because they have the same value :) so "is" better than "==" bu I decided to keep it as it maybe to create a separate task for that?

Reply:
OK, sounds like a separate issue. Would you mind creating it?